src/System.Private.ServiceModel/tools/test/SelfHostWcfService/Certificates/README.md

```diff
@@ -0,0 +1,73 @@
+Certificate generation
+=======================
+
+This file describes how the certificates were created
+and how they are used.  Basic steps are at:
+    https://msdn.microsoft.com/en-US/library/Ff648498.aspx
+
+
+The root certificate authority stored in the "Root" store
+----------------------------------------------------------
+This is a certificate authority used to create the self-signed certificate.
+It was created with these commands from an elevated prompt: 
+
+Template:
+
+```
+    makecert -n "CN=<<CanonicalName>>" -pe -r -sky exchange -cy authority -sk PrivateKey -eku <<Enhanced key usages>> -ss <<Subject cert store>> -sr <<Subject cert store location>> -m <<months to expiry>> <<filename>> 
+```
+
+Actual:
+
+```
+    makecert -n "CN=DO_NOT_TRUST_WcfBridgeRootCA, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf" -pe -r -sky exchange -cy authority -sk PrivateKey -eku 1.3.6.1.5.5.7.3.2,1.3.6.1.5.5.7.3.1 -ss Root -sr LocalMachine -m 2 WcfBridge_RootCA.cer
+```
+
+The certificate gets placed into the *Local Machine*'s *Root Certificate* store, as well as in the current directory's *WcfBridge_RootCA.cer*. 
+
+> *Optional:* If it's necessary to export the private key, you can do it using certmgr.msc:
+> 
```

Comment:
Nit: is it optional for us?  Ideally these steps are a script someone will follow to update the certificates, and they will want to know what steps are mandatory and which are not.